captum/insights/frontend/src/App.css

```diff
@@ -1,354 +1,7 @@
-.app {
-  width: 100%;
-  height: 100%;
-}
-
-.header {
-}
-
-.header__name {
-  font-size: 1.5em;
-  font-weight: bold;
-  padding: 16px 32px;
-  text-transform: uppercase;
-}
-
-.header__nav {
-  border-bottom: solid 1px #ccd0d5;
-  padding-left: 32px;
-  background-color: white;
-  padding-top: 4px;
-}
-
-.header__nav ul {
-  list-style: none;
-  margin: 0;
-  padding: 0;
-}
-
-.header__nav__item {
-  color: #606770;
-  display: inline-block;
-  font-size: 1.2em;
-  line-height: 2em;
-  margin: 0 8px;
-  padding: 0 12px;
-}
-
-.header__nav__item--active {
-  color: black;
-  font-weight: 600;
-  border-bottom: solid 4px #ee4c2c;
-}
-
-.filter-panel {
-  background-color: white;
-}
-
-.filter-panel,
-.viz__panel {
-  align-content: space-between;
-  display: flex;
-  flex-direction: row;
-}
-
-.filter-panel__column {
-  width: 33%;
-  padding: 12px 16px;
-}
-
-.filter-panel__column__title,
-.panel__column__title {
-  font-weight: bold;
-  color: #1c1e21;
-  padding-bottom: 12px;
-}
-
-.filter-panel__column__body {
-  color: #606770;
-}
-
-.filter-panel__column--end {
-  flex-grow: 1;
-  align-self: center;
-  width: auto;
-}
-
-.select {
-  background: none;
-  border: none;
-  border-radius: 0;
-  text-align-last: center;
-  padding: 0 8px;
-  margin: 0 4px;
-  border-bottom: solid 1px #1c1e21;
-  font-size: 1em;
-  appearance: none;
-  color: #1c1e21;
-}
-
-.input {
-  background: none;
-  box-shadow: none;
-  border: none;
-  font-size: 1em;
-  border-bottom: solid 1px #1c1e21;
-  text-align-last: center;
-  padding: 0;
-  margin: 0;
-}
-
-.input--narrow {
-  width: 100px;
-}
-
-.row {
-  display: block;
-}
-
-.row--padding {
-  margin: 8px 0;
-}
-
 button {
   cursor: pointer;
 }
 
```

Comment:
I believe the main purpose of CSS modules is to allow locally scoped styles linked to classnames. For that reason, I put all class-specific styles in the App.module.css file and all global styles in App.css. A CSS selector like this isn't directly accessible by CSS modules since there's no classname tied to it.

Example: if we import the CSS modules like so:
```
import styles from "./App.module.css";
```
You can't refer to `styles.button`, but you can use `styles.btn` since .btn is a classname.

That being said, global styles like button { ... } will still work as expected when placed in a CSS modules file.

Reply:
I'm wondering if we should move this into the .btn class, does it influence the Jupyter notebook at all?

Reply:
I believe the Jupyter notebook buttons are already styled to have `cursor: pointer;`, but I think it would be more proper to scope this locally in the `.btn` class.